docker/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.3'
 services:
   redis:
     image: redis:${REDIS_VERSION:-3.2-alpine}
-    hostname: travis-redis
+    hostname: travis-redis # should we rename ? remove ?
```

Comment:
Did you try to remove the hostname ? If so, does it have an impact on the CI ?

Reply:
There is no impact. This was only to specify this compose file was for CI tests only.

Reply:
Yes we can remove hostnames on each service. I would better change `travis` by `ci`